geopandas/tests/test_geodataframe.py

```diff
@@ -414,9 +418,16 @@ def test_mixed_types_to_file(self):
         tempfilename = os.path.join(self.tempdir, 'test.shp')
         s = GeoDataFrame({'geometry': [Point(0, 0),
                                        Polygon([(0, 0), (1, 0), (1, 1)])]})
-        with pytest.raises(ValueError):
+        with pytest.raises((ValueError, RuntimeError)):
```

Comment:
Where does the `RuntimeError` occur? This seems worth adding a comment in explanation in the test.

Reply:
done